Added.. a lot

ibmua opened the pull request:
Also, made indentation Richard Hendricks-friendly (tabs). Can change it to spaces if need be, though, I had indentation problems when copying your code. Because of these spaces - one can never clearly know where's how many spaces.

ibmua committed:
Added "_act" naming for activation functions, added 

Also, made indentation Richard Hendricks-friendly (tabs). Can change it to spaces if need be, though, I had indentation problems when copying your code. Because of these spaces - one can never clearly know where's how many spaces.

ducha-aiki commented:
@ibmua Thanks.
The reason for spaces - http://stackoverflow.com/a/120956/1983544 so please make it spaces :)

ibmua committed:
Spaced out.. Plus -help added.

While the argument for spaces is completely wrong
 - "The problem with tabs is that they are invisible, and people can never agree on the width of tabs."
 - It's most common, meaningful, default and completely agreeable to use 4-space tabs. It's extremely easy for any moderate computer user to configure his editor to use 4-spaced tabs, it's not even a question for programmers. Meanwhile, it wouldn't hurt absolutely anything if the amount of space a single tab takes is configured differently, so people who edit documents and have, for example, a small screen, are completely free to use less spacing, which is not even possible when using spaces.

As for not seeing where there's a space, where there's a tab, well this picture https://drive.google.com/file/d/0BwFUmoE_MqxfVTdWZTRXVDV2dnM/view?usp=sharing says it all. You can easily see where's what in fine simple free editors like Notepad++.

I won't even go into details in what ways tabs are much better than spaces, because it's too damn obvious. Using tabs makes editing much faster, period. As for PEP 8, as for many major companies using spaces, people get wrong. Standards get wrong. Change over time. I'd hate to work on a company that would require me to use spaces instead.

But that's your project, you have a set such a standard here, so here you go.

ibmua committed:
Update lsuv_init.py

ibmua committed:
Lots of help editing, plus CPU-default now.

ibmua committed:
Update lsuv_init.py

ibmua committed:
Update lsuv_init.py

ducha-aiki commented:
Thanks!
